### ISSUE TITLE
Export as Notebook Results in Blank Notebook

### ISSUE THREAD

**Issue by chlafreniere** [microsoft/azuredatastudio#13227]
Issue Type: <b>Bug</b>

Export as Notebook action results in a blank notebook being created:
- Choose "SELECT TOP 1000" from the context menu after right clicking on a table
- Wait for results to come back from query
- Click Export as Notebook

Azure Data Studio version: azuredatastudio-rc1 1.24.0-rc1 (04117b2333373bf326d367de40fb018a83218775, 2020-11-03T00:46:15.431Z)
OS version: Windows_NT x64 10.0.19042

![image](https://user-images.githubusercontent.com/40371649/98158008-1d8f0180-1e8f-11eb-9622-32154bd9e719.png)

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i7-6700 CPU @ 3.40GHz (8 x 3408)|
|GPU Status|2d_canvas: enabled<br>flash_3d: enabled<br>flash_stage3d: enabled<br>flash_stage3d_baseline: enabled<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>oop_rasterization: disabled_off<br>opengl: enabled_on<br>protected_video_decode: unavailable_off<br>rasterization: enabled<br>skia_renderer: disabled_off_ok<br>video_decode: enabled<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|undefined|
|Memory (System)|63.92GB (36.36GB free)|
|Process Argv||
|Screen Reader|no|
|VM|0%|
</details><details><summary>Extensions (1)</summary>

Extension|Author (truncated)|Version
---|---|---
azdata|Mic|0.3.1

</details>

**Comment by Charles-Gagnon** [comment]
@chlafreniere I can't repro this

Is what you're seeing perhaps caused by this issue : https://github.com/microsoft/azuredatastudio/issues/12922 ? That's still an ongoing problem I run into all the time when using the convert feature

**Comment by Charles-Gagnon** [comment]
@chlafreniere So I played around with this a bit more and I definitely think this is related to whatever bug is causing #12922. I was able to consistently repro this by : 

1. Closing all notebook editors
1. Creating a new Notebook (should be named `Notebook-0`
1. Close it without changing anything - it should close without prompting
1. Open SQL editor and enter some text
1. Click `Export as Notebook`

A new Notebook is opened with the name `Notebook-0` but it's empty

Interestingly enough this doesn't happen if I change step 3 and add something to the Notebook, say a code cell. Then when I close it I get prompted whether I want to save and click No. After this then when the SQL file is exported to a notebook it'll open an editor titled `Notebook-0` but the Notebook will have the content from the SQL file as expected.
